docs/website/mmp-errors.md

```diff
@@ -174,6 +174,7 @@ An unexpected error occured when processing an assembly.
 
 The assembly causing the issue is named in the error message. In order to fix this issue the assembly will need to be provided in a [bug report](https://bugzilla.xamarin.com) along with a complete build log with verbosity enabled (i.e. `-v -v -v -v` in the **Additional mtouch arguments**).
 
+### <a name="MM2104"/>MM2104: Unable to link assembly '{0}' as it is mixed-mode.
```

Comment:
Worth a bit more description, maybe a link to MSDN that describe what is a mixed-mode assembly ?